torch/_dynamo/output_graph.py

```diff
@@ -232,9 +232,7 @@ def __init__(
                 allow_scalar_outputs=config.capture_scalar_outputs,
                 allow_dynamic_output_shape_ops=config.capture_dynamic_output_shape_ops,
                 frame_id=frame_state["_id"],
-            )
-            if config.dynamic_shapes
-            else None,
+            ),
```

Comment:
Just curious - why? :)

Reply:
We're going to switch `config.dynamic_shapes` to true by default, and so making local changes like this help us see what would break if we do it.